### ISSUE TITLE
[dapps] Show CurrentBalance in dApps

### ISSUE THREAD

**Issue by Alonski** [counterfactual/monorepo#1210]
The current architecture was built with the fact that the PG UI shows the `CurrentBalance` of the `User` at all times while the dApps live inside an iFrame.

With the new architecture the PG UI lives inside of MM, which is usually closed or in a different tab, and the dApp lives standalone. 

This creates a UX issue where a person playing/using a dApp doesn't know their `CurrentBalance` of their funds in the `multisig`

This issue suggests adding UX for `CurrentBalance` in the dApps: High Roller and Tic Tac Toe

**Comment by snario** [comment]
Is there any work to be done in the `Node`? No, right? Because we already have a method you can call on the Node to get this value.

**Comment by Alonski** [comment]
@snario that is correct.
We are also already requesting the balance in the dApp to make sure that we can proposeInstall.

We just need to show it :)

**Comment by emansipater** [comment]
Just in terms of communicating consistently to the user, I think we want to move towards a picture where the user's *channel* balance is just thought of as part of their wallet balance.  The UI for a specific *app* needs to show the money committed to that app, and they do need to know their channel "limit" when opening an app. But we should figure out a clear way of communicating all this consistently so the user doesn't get confused with all their different balances.  I'll add it to the "vocabulary systematization" project.

**Comment by Alonski** [comment]
@emansipater I agree.
Currently the user can open a dApp with 0.1 ETH in *channel* balance.
They then play 10 games over the course of an hour and lost every time.
They then try to play again and get an error. They forgot how much they played.

Since MM UI is usually hidden and transactions don't need to be signed. Users of a dApp can quickly lose track of their balance.

I think we should teach that best practice for State Channel dApps is to include the balance in the UI.

**Comment by snario** [comment]
This is partially being addressed by @ebryn in his Wallet UI spike and @cf19drofxots with the work being done with Connext.

**Comment by Alonski** [comment]
@snario Can you elaborate on how this is being addressed?
Currently, playing High Roller with the MM integration. It is very easy to lose track of your state channel balance.

**Comment by snario** [comment]
Makes sense. Can we subdivide this into two issues then, one for TTT and one for High Roller? They need to call the `GET_FREE_BALANCE_STATE` endpoint.
